diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e8c76c..be5f4ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -165,6 +165,8 @@ if(L4T_FOUND)
         if(${L4T_REVISION} GREATER_EQUAL "3")
             add_subdirectory(gst-zedxone-src)
         endif()
+    elseif(${L4T_RELEASE} GREATER "36")
+        add_subdirectory(gst-zedxone-src)
     endif()
 endif()
 add_subdirectory(gst-zed-demux)
